client/js/sign.js

```diff
@@ -0,0 +1,70 @@
+const inForm = document.getElementById('signin-frm');
+const upForm = document.getElementById('signup-frm');
+
+// POST data and get response to use the result for user check
+async function postData(url = '', data = {}) {
+  const response = await fetch(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    redirect: 'follow',
+    body: JSON.stringify(data)
+  });
+
+  const status = response.status;
+  const res = await response.json();
+  return { status: status, response: res }
+}
+
+// Check user existence
+inForm.addEventListener('submit', (frm) => {
```

Comment:
[nit] the parameter for this is conventionally known as `e` or `event`.  It can honestly be labeled anything but it might be better to follow convention: https://www.w3schools.com/js/js_htmldom_eventlistener.asp

Reply:
There is no e / element outlined in the URL but I get your point and changed them.

Reply:
It's under the `Syntax` section:

`element.addEventListener(event, function, useCapture);`

Reply:
I see.
I didnt see e and thought this could be anything as other function like (err, row)
Thanks ! Appreciate for the reference:)